docs/helpers.asciidoc

```diff
@@ -466,12 +490,16 @@ for (const doc of documents) {
 
 
 [discrete]
-=== Scroll Search Helper
+[[scroll-search-helper]]
+=== Scroll search helper
 
 ~Added~ ~in~ ~`v7.7.0`~
 
-This helpers offers a simple and intuitive way to use the scroll search API. Once called, it returns an https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of[async iterator] which can be used in conjuction with a for-await...of. +
-It handles automatically the `429` error and uses the client's `maxRetries` option.
+This helpers offers a simple and intuitive way to use the scroll search API. 
+Once called, it returns an 
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of[async iterator] 
```

Comment:
I've noticed this link gets encoded strangely. The three dots get substituted with an ellipsis which then results in a 404: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await%E2%80%A6%E2%80%8Bof

My knowledge of Asciidoc is quite low unfortunately, but would the [`link:` macro](https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/) resolve it?

Reply:
Thanks for spotting! Unfortunately, `link:macro` won't resolve it correctly. I change the URL to the URL of the async function page as link text refers to that one anyway.